init.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+echo what is your PROJECT_NAME?
+read PROJECT_NAME
+cd ..
+mv web-boilerplate $PROJECT_NAME
+cd $PROJECT_NAME
+rm -rf .git
+git init .
+
+echo "# $PROJECT_NAME
```

Comment:
Hmm. I like the idea of building up a readme for the new project. I think a better way to do it though would be to have a `README_TEMPLATE.md` file, that you copy into place with `rm README.md && mv README_TEMPLATE.md README.md`

Reply:
@dkniffin Is there an easy to interpolate values like this in markdown?  I really like the idea of building the readme like this where we can set stuff like the project name

Reply:
You could do something with sed https://unix.stackexchange.com/questions/159367/using-sed-to-find-and-replace

Reply:
The reason I kept it in the script itself was so that there's less cleanup if you didn't go with the script. I considered writing it so that if you tried to run it after changing it away from `web-boilerplate` that it would just delete itself.

Reply:
I'm wondering why anyone would actually want to do this manually, now that we have a script

Reply:
good point.